spec/models/form/sales/pages/nationality1_spec.rb

```diff
@@ -1,22 +1,20 @@
 require "rails_helper"
 
 RSpec.describe Form::Sales::Pages::Nationality1, type: :model do
-  subject(:page) { described_class.new(page_id, page_definition, subsection) }
+  subject(:page) { described_class.new(nil, nil, subsection) }
 
-  let(:page_id) { nil }
-  let(:page_definition) { nil }
   let(:subsection) { instance_double(Form::Subsection) }
 
   it "has correct subsection" do
-    expect(page.subsection).to eq(subsection)
+    expect(page.subsection).to be subsection
   end
 
   it "has correct questions" do
-    expect(page.questions.map(&:id)).to eq(%w[national othernational])
+    expect(page.questions.map(&:id)).to eq %w[national]
```

Comment:
Any reason for changing the syntax?

Reply:
you mean missing out the parentheses? Just less typing, equal or improved readability

Reply:
I'd keep it consistent with the rest of the codebase, plus less changes when reviewing